UnicodeJsps/src/main/java/org/unicode/jsp/CachedProps.java

```diff
@@ -44,7 +44,7 @@
     final BiMultimap<String,String> nameToAliases = new BiMultimap<String,String>(null,null);
     final Map<String,BiMultimap<String,String>> nameToValueToAliases = new LinkedHashMap();
 
-    static CachedProps CACHED_PROPS = getInstance(VersionInfo.getInstance(12));
```

Comment:
I'm surprised at this; if based on master it would be replacing 13 by 14.

Reply:
Is this a bug? I hadn't changed this value. Should it be calculated ?

Reply:
Lemme check.

Reply:
Yes, that should be the version of the beta props. I think it is built that way so that it doesn't pull in the BIN properties if BETA is off. For now, let's just leave it at 14, but file an issue.